### DISCUSSION TITLE
SSO MFA

### DISCUSSION THREAD

**Discussion by leyoda** [sinamics/ztnet#436] [General]
Hi
Is it possible to associate MFA with ZTNET ZT users so that they can connect? 
Best regards,
Lionel

**Comment by sinamics** [comment]
ztnet does not support 2FA at the moment.

**Comment by PizzaProgram** [comment]
I guess this changed since last 8 month ?

![kép](https://github.com/user-attachments/assets/0c1697ce-2dc0-48b4-8071-1faa6bfaf29c)


- Is it safe to activate? 
- What happens if I loose my phone? (How to reset on the server side?)
